src/app/pages/system/advanced/advanced-settings.component.ts

```diff
@@ -446,6 +450,16 @@ export class AdvancedSettingsComponent implements OnInit, AfterViewInit {
           title: helptextSystemAdvanced.fieldset_sessions,
           tableConf: this.sessionsTableConf,
         },
+        {
+          title: helptextSystemAdvanced.fieldset_token,
+          id: AdvancedCardId.Token,
+          items: [
+            {
+              label: this.translate.instant('Token Lifetime'),
+              value: this.window.localStorage.getItem('lifetime'),
+            },
+          ],
+        },
```

Comment:
Also I'd prefer to avoid having a new separate section for just 1 item. Can we combine it with `Sessions` card?
Perhaps something like this:
```
Sessions
Token Lifetime: 300 seconds          [Configure]

Active sessions
- Active sessions table -
[ Terminate other sessions ]

```